src/kafka/values.yaml

```diff
@@ -127,6 +127,13 @@ persistence:
   ##
   mountPath: /opt/kafka/data  # TODO
 
+## Mounting secrets. Useful for JAAS/Kerberos.
+##
+secrets:
```

Comment:
^ since this must be singular in present design, please change `secrets:` to `secret:`

Reply:
Done.